sphinx_needs/data.py

```diff
@@ -28,10 +28,15 @@ class NeedsFilterType(TypedDict):
     status: list[str]
     tags: list[str]
     types: list[str]
-    result: list[str]
```

Comment:
I removed the result from captured data, because his could greatly increase the size of in-memory data for large projects